build.gradle.kts

```diff
@@ -81,6 +85,11 @@ dependencies {
     testImplementation("org.mockito:mockito-junit-jupiter:${project.extra["mockitoVersion"]}")
     testImplementation("org.apiguardian:apiguardian-api:1.1.2") // https://github.com/gradle/gradle/issues/18627
 
+    testImplementation("org.apache.spark:spark-core_2.13:${project.extra["sparkVersion"]}")
+    testImplementation("org.apache.spark:spark-sql_2.13:${project.extra["sparkVersion"]}")
+    testImplementation("org.apache.spark:spark-catalyst_2.13:${project.extra["sparkVersion"]}")
+    testImplementation("org.apache.spark:spark-streaming_2.13:${project.extra["sparkVersion"]}")
```

Comment:
```suggestion
    testImplementation("org.apache.spark:spark-streaming_${project.extra["scalaBinaryVersion"]}:${project.extra["sparkVersion"]}")
```

Reply:
I've integrated your suggestion + cross build with conditional sources in function of scala212 and greater.

Now need to add build of both version to have artifact properly added to nexus